File: ApiRequest.cs
using System;
using System.Net.Http;
using System.Threading.Tasks;

class Program
{
    static async Task Main(string[] args)
    {
        Console.Write("Enter your input: ");
        string userInput = Console.ReadLine();

        string apiUrl = $"https://endpoint.spamtec.cc/v1/tool?key=key&input={Uri.EscapeDataString(userInput)}";
        
        using (HttpClient client = new HttpClient())
        {
            try
            {
                HttpResponseMessage response = await client.GetAsync(apiUrl);
                response.EnsureSuccessStatusCode();
                string responseBody = await response.Content.ReadAsStringAsync();

                Console.WriteLine("API response:");
                Console.WriteLine(responseBody);
            }
            catch (HttpRequestException e)
            {
                Console.WriteLine($"Error: {e.Message}");
            }
        }
    }
}

File: ApiRequest.kt
//Save the script as ApiRequest.kt and compile it using the following command:
//kotlinc ApiRequest.kt -include-runtime -d ApiRequest.jar
//java -jar ApiRequest.jar

import java.io.BufferedReader
import java.io.InputStreamReader
import java.net.HttpURLConnection
import java.net.URL

fun main() {
    print("Enter your input: ")
    val userInput = readLine()

    val apiURL = "https://endpoint.spamtec.cc/v1/tool?key=key&input=$userInput"
    val url = URL(apiURL)
    val connection = url.openConnection() as HttpURLConnection

    try {
        connection.requestMethod = "GET"
        val inputStream = BufferedReader(InputStreamReader(connection.inputStream))
        val response = inputStream.readText()
        inputStream.close()

        println("API response:")
        println(response)
    } catch (e: Exception) {
        println("Error making API request: $e")
    } finally {
        connection.disconnect()
    }
}
